lib/galaxy/tool_util/parser/yaml.py

```diff
@@ -40,6 +40,13 @@ def parse_edam_operations(self):
     def parse_edam_topics(self):
         return self.root_dict.get("edam_topics", [])
 
+    def parse_xrefs(self):
+        xrefs = []
+        for i, xref_dict in enumerate(self.root_dict.get("xrefs", [])):
+            if xref_dict["type"] == "bio.tools":
+                xrefs.append(xref_dict["value"])
+        return xrefs
```

Comment:
I think this method should also return a list of dicts like what I've suggested for lib/galaxy/tool_util/parser/xml.py .